src/App.svelte

```diff
@@ -51,8 +52,68 @@
       link: "https://github.com/EddieJaoudeCommunity/hacktoberfest-practice",
     }
   ];
-</script>
 
+  const Moderators_Data = [
```

Comment:
this is not a blocker, but could you follow the same convention for variable names as the one above (`cardData`), which is camel case?

Reply:
Vs code added it I guess will remove it 😅